synchronizer/l1_rollup_info_producer_test.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func Test_L1Get(t *testing.T) {
+func TestExploratoryL1Get(t *testing.T) {
```

Comment:
Why is it skipped?

Reply:
It's just an exploratory test, It's used to execute code and debug it, check the current behaviour,... etc..